### DISCUSSION TITLE
Same project in two different windows ?

### DISCUSSION THREAD

**Discussion by PierreMarchand20** [alefragnani/vscode-project-manager#665] [Q&A] [answered]
Naive question, but I could not find the answer (I probably use the wrong keywords): is this possible to have the same project in two different windows ? 

It is useful if you have multiple screens for example.

**Comment by alefragnani** [comment, marked as answer]
Hi @PierreMarchand20 ,

VS Code itself doesn's support this, so you won't be able to do this via extension as well.

But, I just noticed that recently, VS Code added a new `Workspaces: Duplicate as Workspace in New Window` command, which may be way to accomplish what you need.

Hope this helps

**Reply by PierreMarchand20** [reply]
It does ! thank you for your answer AND you extension ;-)
